docs/okeps/okep-50088-evpn.md

```diff
@@ -0,0 +1,439 @@
+# OKEP-5088: EVPN Support
+
+* Issue: [#5088](https://github.com/ovn-org/ovn-kubernetes/issues/5088)
+
+## Problem Statement
+
+The purpose of this enhancement is to add support for EVPN within the OVN-Kubernetes SDN, specifically with BGP. This
+effort will allow exposing User Defined Networks (UDNs) externally via a VPN to other entities either
+inside, or outside the cluster. BGP+EVPN is a common and native networking standard that will enable integration into
```

Comment:
Not sure if we need to mention around how this design aligns with upcoming OVN native EVPN support to avoid API/workflow conflicts.

Reply:
its outside the scope of this enhancement. Hopefully @dceara reads this thing and ensures compatibility ;)

Reply:
@trozet I'll try to but I think the right approach might be to sit down together with members of both communities to align on the right use cases.  Maybe OVS+OVN'25 conference is an appropriate setting for that.